cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -32,10 +35,18 @@ function deploy_sriov_operator {
   popd
 }
 
+if [[ -z "$(kubectl get nodes | grep $FIRST_WORKER_NODE)" ]]; then
+  SRIOV_NODE=$MASTER_NODE
+else
+  SRIOV_NODE=$FIRST_WORKER_NODE
+fi
+
+SRIOV_NODE_CMD="docker exec -it -d ${SRIOV_NODE}"
```

Comment:
Nit: maybe move this to line 77 to have it more adjacent to the usage.

Reply:
Done, do you mind re-approving this?